osutil/cp_linux_test.go

```diff
@@ -36,11 +36,25 @@ func (s *cpSuite) TestCpMulti(c *C) {
 	c.Check(s.f2, testutil.FileEquals, s.data)
 }
 
+func (s *cpSuite) getReadOnlyFd(c *C) *os.File {
+	c.Check(osutil.CopyFile(s.f1, s.f2, osutil.CopyFlagDefault), IsNil)
+	c.Check(os.Chmod(s.f2, 0444), IsNil)
+	f2, err := os.Open(s.f2)
+	c.Assert(err, IsNil)
+	return f2
+}
+
 func (s *cpSuite) TestDoCpErr(c *C) {
-	f1, err := os.Open(s.f1)
+	src, err := os.Open(s.f1)
 	c.Assert(err, IsNil)
-	st, err := f1.Stat()
+	defer func() { src.Close() }()
```

Comment:
For simple (one liner, no error handling, etc) deferred statements, this shorthand is a bit nicer to read
```suggestion
	defer src.Close()
```